collect_app/src/androidTest/java/org/odk/collect/android/storage/StorageMigrationDialogTest.java

```diff
@@ -0,0 +1,80 @@
+package org.odk.collect.android.storage;
+
+import android.Manifest;
+
+import androidx.test.espresso.intent.rule.IntentsTestRule;
+import androidx.test.rule.GrantPermissionRule;
+
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.RuleChain;
+import org.odk.collect.android.activities.MainMenuActivity;
+import org.odk.collect.android.support.CopyFormRule;
+import org.odk.collect.android.support.ResetStateRule;
+import org.odk.collect.android.support.StorageMigrationNotPerformedRule;
+import org.odk.collect.android.support.pages.MainMenuPage;
+import org.odk.collect.android.support.pages.StorageMigrationDialogPage;
+
+public class StorageMigrationDialogTest {
```

Comment:
I'm wondering if it might make sense to combine these tests with the `StorageMigrationBannerTest` as one file that tests the migration flow.

Reply:
Done.